src/resources/i18n/translations/en.json

```diff
@@ -15,13 +15,13 @@
   "Account": "Account",
   "Active Downloads": "Active Downloads",
   "Actual amount will be higher due to network fees": "The actual amount also includes network fees",
-  "Add current item next in queue when playing a new item": "Add current item next in queue when playing a new episode",
-  "Add Custom RSS Feed": "Add Custom RSS Feed",
+  "Add current item next in queue when playing a new item": "Immediate playback",
+  "Add Custom RSS Feed": "Add custom RSS feed",
   "Add custom RSS feeds": "Add custom RSS feeds",
   "Add to Playlist": "Add to Playlist",
   "Add Wallet": "Add Wallet",
   "AddPodcastByRSSScreenText1": "Note: Clips, playlists, and sharing are not supported for custom RSS feeds.",
-  "AddPodcastByRSSScreenText2": "Use the \"Request podcast link below to add a podcast with support for clips and playlists to our database.",
+  "AddPodcastByRSSScreenText2": "Click the link below to add a podcast with support for clips and playlists to our database.",
```

Comment:
This should ideally be a "Add a podcast with support for clips and playlists to our database." link itself. ?

Reply:
I like it 👍